CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing Guide
+
+Thank you for your interest in contributing to Enzyme.jl! There are multiple ways to contribute, and we appreciate all contributions. In case you have questions, you can either use Enzyme.jl's very own [Slack Channel](https://join.slack.com/t/julialang/shared_invite/zt-w0pifg7p-18IUSkZy_WpofNumiTTROQ), the [Julia Discourse](https://discourse.julialang.org/).
```

Comment:
```suggestion
Thank you for your interest in contributing to Enzyme.jl! There are multiple ways to contribute, and we appreciate all contributions. In case you have questions, you can either use Enzyme.jl's `#enzyme` [Slack Channel](https://julialang.org/slack/), or the [Julia Discourse](https://discourse.julialang.org/).
```